src/types/item-itunes.ts

```diff
@@ -2,7 +2,7 @@ export interface ItemITunes {
   /** (iTunes specific) author of the podcast */
   itunesAuthor?: string;
   /** (iTunes specific) specifies if the podcast contains explicit content */
-  itunesExplicit?: boolean;
+  itunesExplicit?: boolean | ('yes' | 'no' | 'clean');
```

Comment:
Why do you not also use the new type `FeedItunesExplicit` here?

Reply:
I was following what I thought was the convention.
My preferred solution is to create an `ItunesExplicit` type and use that in both places. Better name than `FeedItunesExplicit`